apps/console/src/features/applications/components/settings/access-configuration.tsx

```diff
@@ -560,6 +578,52 @@ export const AccessConfiguration: FunctionComponent<AccessConfigurationPropsInte
         }
     };
 
+    /**
+     * Handles Authenticator delete button on click action.
+     *
+     * @param e - Click event.
+     * @param name - Protocol name.
+     */
+    const handleProtocolDeleteOnClick = (e: MouseEvent<HTMLDivElement>, name: string): void => {
+        if (!name) {
+            return;
+        }
+
+        const deletingProtocol: string = inboundProtocols
+            .find((protocol: string) => protocol === name);
+
+        if (!deletingProtocol) {
```

Comment:
Let's move these type of guard conditions to the top of the function body for readability.

Reply:
Since the condition is using the `deletingProtocol ` variable value, condition cannot be moved to top further